src/core/a-animation.js

```diff
@@ -463,3 +479,35 @@ function strToBool (str) {
 function boolToNum (bool) {
   return bool ? 1 : 0;
 }
+
+/**
+ * Converts a number 0-255 to hex
+ * @param {color}
+ * @returns {string}
+ */
+function componentToHex (color) {
+  var hex = color.toString(16);
```

Comment:
@dmarcos @ngokevin Hey this could be clamped in the function that calls this by doing a:
`  return Math.floor(Match.min(Math.abs(color), 255) * 255);`
inside of the convertToIntegerColor() function.  Would this be acceptable?  Or would you prefer a separate clamp function?